core/common/src/thrift/exception.thrift

```diff
@@ -117,5 +120,5 @@ enum TExceptionStatus {
 
 exception AlluxioTException {
   2: string message
-  4: TExceptionStatus status
+  4: TStatus status
```

Comment:
we used them for the class name and alluxio error code in the past, I think it's a little better not to reuse numbers